src/client/pages/OfferDebugger/components/QuoteData.tsx

```diff
@@ -374,6 +381,7 @@ export const QuoteData: React.FC<OfferProps> = ({ quoteCartId }) => {
 
               <Button
                 type="submit"
+                size="lg"
```

Comment:
That's a big button 😄 nit: let's have the `Go to offer page` button in the same size :)

Reply:
ahhh good point!

I realized that Guilherme was clicking "create new quote cart" when he meant to click "create quote" so I wanted to make on of them stand out.